internal/services/storage/storage_account_resource.go

```diff
@@ -1563,6 +1571,20 @@ func resourceStorageAccountUpdate(d *pluginsdk.ResourceData, meta interface{}) e
 
 	}
 
+	if d.HasChange("sftp_enabled") {
+		sftpEnabled := d.Get("sftp_enabled").(bool)
+
+		opts := storage.AccountUpdateParameters{
+			AccountPropertiesUpdateParameters: &storage.AccountPropertiesUpdateParameters{
+				IsSftpEnabled: &sftpEnabled,
+			},
+		}
+
+		if _, err := client.Update(ctx, id.ResourceGroup, id.Name, opts); err != nil {
+			return fmt.Errorf("updating Azure Storage Account sftp_enabled %q: %+v", id.Name, err)
```

Comment:
fwiw this can become:

```suggestion
			return fmt.Errorf("updating `sftp_enabled` for %s: %+v", *id, err)
```

which'll make this:

"updating `sftp_enabled` for Storage Account "SomeAccount" (Resource Group "SomeGroup"): some error"

Reply:
resolved